pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dev_dependencies:
   flutter_launcher_icons: "^0.6.0"
   build_runner: ^0.9.0
   json_serializable: ^0.5.4
+  mockito: ^3.0.0
```

Comment:
Actually mockito is not used in this PR but I'm sure this helpful to write tests.